versioned_docs/version-latest/protocol/pulsar.mdx

```diff
@@ -469,7 +404,111 @@ Initial plotting for plot: #0 (/home/username/.local/share/pulsar/farms)
+
+- `30333`
+- `30433`
+- `30533`
+
+On the desktop side if you have a router in front of your computer, you'll need to forward TCP ports to the machine on which your node is running (how this is done varies from router to router, but there is always a feature like this, refer to [How to Forward Ports](port-forwarding.md) for a more in-depth tutorial).
+If you're connected directly without any router, then again nothing needs to be done in such case.
+
+
+### View your Logs
+
+A good place to start if you are facing trouble is by viewing your logs and seeing if there are any errors or insights that might be available. You can find the location for your logs below
+
+<Tabs groupId="OS">
+<TabItem value="windows" label="🖼️ Windows" default>
+
+Your Logs will be found in `%USERPROFILE%/AppData/Local/pulsar/logs`
+
+</TabItem>
+
+<TabItem value="macos" label="🍎macOS">
+
+Your Logs will be found in `$HOME/Library/Logs/pulsar/`
+
+</TabItem>
+
+<TabItem value="linux" label="🐧Ubuntu">
+
+Your Logs will be found in `$HOME/.local/share/pulsar/logs`
+
+</TabItem>
+</Tabs>
+
+
+### Wipe Node & Farmer
+
+:::tip Updated from a previous version and now having issues?
+
+Occasionally after updating to a new version of the Pulsar you will need to wipe your node and farmer, generally this should not be required but can be attempted if your farmer is having issues after having had worked fine previously.
+
+:::
+
+To simply restart the node, go to the terminal where you started the `farm` command, and press `Ctrl + C` you should see a shutdown message appear and the application will attempt a simple shutdown, if you dont see the message press `Ctrl + C` again to force shutdown. You can then simply start the farmer again with the `farm` command you used prior.
+
+Use the same file name as the previous init and farm steps, then add the wipe command to free the previous storage that was being used. Generally, only do this if you have severe errors and are prompted by a staff member.
+
+```shell-session
+./pulsar-file-name wipe
+```
+
+After wiping, follow the `init` and `farm` steps above to start farming again!
+
+
+:::info Finding your settings.toml
+
+After running `pulsar init`, the prompt will display where the `settings.toml` is generated. However in case you missed it, you can find the file based on your operating system:
+
+<Tabs groupId="OS">
+<TabItem value="windows" label="🖼️ Windows" default>
+
+Your `settings.toml` will be found in `$FOLDERID_RoamingAppData/pulsar/settings.toml`
+
+</TabItem>
+
+<TabItem value="macos" label="🍎macOS">
+
+Your `settings.toml` will be found in `$HOME/Library/Application Support/pulsar/settings.toml`
+
+</TabItem>
+
+<TabItem value="linux" label="🐧Ubuntu">
+
+Your `settings.toml` will be found in `$HOME/.config/pulsar/settings.toml`
+
+</TabItem>
+</Tabs>
+
+:::
+
+
+
+## Still having trouble with your farmer?
```

Comment:
Let's make it a subsection of troubleshooting by using ### instead.

Reply:
Nice 👍